src/DataExplorer/DataExplorer.jsx

```diff
@@ -1,10 +1,59 @@
 import React from 'react';
+import DataSummaryCardGroup from '../components/DataSummaryCardGroup';
+import SummaryChartGroup from '../components/charts/SummaryChartGroup';
+import PercentageStackedBarChart from '../components/charts/PercentageStackedBarChart';
+
+const countItems = [
```

Comment:
don't hard code in the component, you should have some list parameters and the way to load data into that component.

Reply:
Oh this is just dummy data here~ we are developing this page~

Reply:
even you are developing the page. Don't use hardcode data inside the component @qingyashu